OptiFineDoc/assets/minecraft/optifine/lang/pl_pl.lang

```diff
@@ -584,3 +711,11 @@ of.options.SCREENSHOT_SIZE.tooltip.3=  2x-4x - własny rozmiar
 of.options.SCREENSHOT_SIZE.tooltip.4=Zapisywanie większych zrzutów może wymagać więcej pamięci,
 of.options.SCREENSHOT_SIZE.tooltip.5=opcja nie jest kompatybilna z szybkim renderowaniem ani
 of.options.SCREENSHOT_SIZE.tooltip.6=antyaliasingiem. Wymaga wsparcia bufora ramki GPU.
+
+of.options.SHOW_GL_ERRORS=Pokazuj błędy GL
+of.options.SHOW_GL_ERRORS.tooltip.1=Pokazuj błędy OpenGL
+of.options.SHOW_GL_ERRORS.tooltip.2=Kiedy aktywne, błędy OpenGL są pokazywane na czacie.
+of.options.SHOW_GL_ERRORS.tooltip.3=Wyłącz tą opcję tylko jeżeli jest znany konflikt
```

Comment:
Poprawka (gramatyka):

of.options.SHOW_GL_ERRORS.tooltip.3=Wyłącz tę opcję tylko jeżeli jest znany konflikt